tiny_tokenizer/word_tokenizer.py

```diff
@@ -21,36 +23,66 @@ def __init__(self, tokenizer=None, flags=''):
             self.tokenizer = natto.MeCab(flags)
             self.tokenize = self._mecab_tokenize
 
+        elif tokenizer == 'Sentencepiece':
+            import sentencepiece
+            self.tokenizer = sentencepiece.SentencePieceProcessor()
+            self.tokenizer.load(flags)
+            self.tokenize = self._sentencepiece_tokenize
+
+        elif tokenizer == 'Character':
+            self.tokenize = self._character_level_tokenizer
+
         else:
             warnings.warn('Return input directly')
             self.tokenizer = None
             self.tokenize = lambda x: x
 
     def _mecab_tokenize(self, sentence):
-        """
-        :param sentence: a sentence to be tokenized
-        :type sentence: str
-        :return: a tokenized sentence words
-                 are segmented with whitespace
-        :rtype: str
+        """Mecab tokenizer.
+
+        Arguments:
+            sentence {str} -- raw sentence
         """
         return self.tokenizer.parse(sentence)
 
     def _kytea_tokenize(self, sentence):
-        """
-        :param sentence: a sentence to be tokenized
-        :type sentence: str
-        :return: a tokenized sentence words
-                 are segmented with whitespace
-        :rtype: str
+        """Kytea tokenizer.
+
+        Arguments:
+            sentence {str} -- raw sentence
         """
         return ' '.join(self.tokenizer.getWS(sentence))
 
+    def _sentencepiece_tokenize(self, sentence):
+        """Sentencepiece tokenizer.
+
+        Arguments:
+            sentence {str} -- raw sentence
+        """
+        return ' '.join(self.tokenizer.EncodeAsPieces(sentence))
+
+    def _character_level_tokenizer(self, sentence):
```

Comment:
Other methods are `_mecab_tokenize`, `_kytea_tokenize`, and `_sentencepiece_tokenize`. How about `_character_level_tokenize`?

Reply:
You're right, thanks!